spec/paths/kyc-documents@{id}@review.yaml

```
@@ -0,0 +1,36 @@
+parameters:
+  - $ref: "#/components/parameters/resourceId"
+patch:
+  tags:
+    - KYC Documents
+  summary: Review a KYC document
+  operationId: PatchKycDocument
```

```suggestion
  operationId: PostKycDocumentReview
```

Please leave `PatchKycDocument` reserved for PATCH to parent endpoint.